p2p_wallet/Services/Repositories/BaseRepository/ListRepository/ListRepository.swift

```diff
@@ -27,63 +26,39 @@ class ListRepository<ItemType: Hashable>: ItemRepository<[ItemType]> {
     // MARK: - Actions
 
     /// Erase data and reset repository to its initial state
-    override func flush() {
+    override func reset(autoFetch: Bool = true) {
         paginationStrategy?.resetPagination()
-        super.flush()
-    }
-    
-    /// Erase and reload all data
-    override func reload() {
-        super.reload()
+        super.reset(autoFetch: autoFetch)
     }
     
-    /// Refresh data
-    override func refresh() {
```

Comment:
refresh function is needed in case we want reload without erasing data. Like in Home screen